legofy/__init__.py

```diff
@@ -38,6 +39,15 @@ def apply_effect(image, overlay_red, overlay_green, overlay_blue):
 
     return Image.merge(image.mode, channels)
 
+def overlay_effect(color, overlay):
+    '''Actual overlay effect function'''
+    if (133 - color) > 100:
```

Comment:
This can be simplified to `color < 33`
The same logic can be applied to the other ifs.
We are not really passing a color but rather the intensity of Red, Green or blue.
Consider renaming color to intensity.